nubia/internal/interactive.py

```diff
@@ -97,9 +97,14 @@ def _build_cli(self):
             accept_action=AcceptAction.RETURN_DOCUMENT,
         )
 
+        editor = getattr(EditingMode,
```

Comment:
Could you please add some comments here WRT `VI` and `EMACS` constants so a stranger could understand a magic behind `getattr` and `upper()`?

Reply:
Done :)